gaia-header.js

```diff
@@ -123,6 +124,7 @@ module.exports = component.register('gaia-header', {
     debug('attached');
     this.runFontFitSoon();
     this.observerStart();
+    window.addEventListener('resize', this);
```

Comment:
I'm just afraid to make it wrong and end up with not removing the event listener. At least with this form I'm sure it's correct ;)

Where should I add `this.onResize = this.onResize.bind(this)` ? in `created` ? Remember the original onResize is on the prototype while the new one will be on the new instance.

What do you think?